source/framework/tools/src/TRestTools.cxx

```diff
@@ -712,8 +755,18 @@ std::string TRestTools::DownloadRemoteFile(string url) {
         return Replace(url, "local:", "");
     } else {
         string fullpath = REST_USER_PATH + "/download/" + purename;
+        int out;
+        int attempts = 10;
+        do {
+            out = TRestTools::DownloadRemoteFile(url, fullpath);
+            if (out == 1024) {
+                warning << "Retrying download in 5 seconds" << endl;
+                sleep(5);
```

Comment:
Please use `std::this_thread::sleep_for(std::chrono::seconds(5));` instead of `sleep`. Perhaps you migh have to include some libraries such as `#include <chrono>` and  `#include <thread>`

Reply:
done